python/ray/serve/batching.py

```diff
@@ -304,6 +318,47 @@ def get_max_batch_size(self) -> int:
     def get_batch_wait_timeout_s(self) -> float:
         return self.batch_wait_timeout_s
 
+    def _get_curr_iteration_start_time(self) -> float:
+        """Gets current iteration's start time on default _BatchQueue implementation.
+
+        Returns -1 if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "curr_iteration_start_time"):
+            return self.queue.curr_iteration_start_time
+        else:
+            return -1
```

Comment:
why not `None`?

Reply:
`time.time()` can never be `-1`, and returning `-1` simplifies the type hints. I changed it to `None` to match `_get_batching_stack`.